libcodechecker/log/log_parser.py

```diff
@@ -810,28 +821,85 @@ def parse_log(compilation_database,
                           targets, default standard version).
     """
     try:
-        filtered_build_actions = set()
+        uniqued_build_actions = dict()
+        NONE = 0 # Full Action text
```

Comment:
Python3 has enums until that you can create a class like this:
```
class CompileActionUniqueingType(object):
    NONE = 0
    SOURCE_ALPHA = 1
    ...

x = CompileActionUniqueingType.NONE
```

Would be the buildaction module a better place for this type? In that case maybe the name `UniqueingType` would be enough because you would see it from the import that it belongs to a buildaction.